### PR TITLE
[feature] cmd=% work with spaced paths

### PR DESCRIPTION
**Pull request by m10090**
this should allow cmd =% use spaced paths
(sorry for any imperfection this is my frist contribute)

### CODE REVIEW

File: lua/toggleterm/commandline.lua [hunk 1]
```diff
@@ -93,8 +93,13 @@ local term_exec_options = {
 
     for _, path in ipairs(paths) do
       -- solve space in file names
-      local glob_str = path.gsub(" ", "\\ ")
+      if path != ''
```
Comment (akinsho) [inline lua/toggleterm/commandline.lua:96]: This isn't valid `lua` you probably meant to write `~=`

File: .DS_Store
Comment (akinsho) [inline .DS_Store:1]: This shouldn't be commited

File: lua/.DS_Store
Comment (akinsho) [inline lua/.DS_Store:1]: This is also not necessary

**Review by akinsho — CHANGES_REQUESTED** [review]
Thanks for the PR @m10090 would be good to have a clear description in the README of what this PR fixes but I think it looks ok as a change but some things need to be tidied up

File: lua/toggleterm/commandline.lua (hunk 1 above)
Comment (m10090) [reply lua/toggleterm/commandline.lua:96]: Sorry about the inconvenience and I am happy to be a part of your project

**Pull request closed without merging**